common/repo/GitRepo.java

```diff
@@ -365,23 +302,47 @@ public void removeBranch(String branch) {
   @Override
   public String getFileContents(String commitId, String path) {
     CommandResult commandResult = runCommand("show " + commitId + ":" + path);
-    String result = commandResult.stdout;
-    int lastIndexOfNewLineSymbol = result.lastIndexOf("\n");
-    if (lastIndexOfNewLineSymbol >= 0) {
-      result =
-          new StringBuilder(result)
-              .replace(lastIndexOfNewLineSymbol, lastIndexOfNewLineSymbol + 1, "")
-              .toString();
-    }
-    return result;
+    return commandResult.stdout;
```

Comment:
You can simplify to:
`return runCommand("show " + commitId + ":" + path).stdout;`